### DISCUSSION TITLE
a little questions about bumble

### DISCUSSION THREAD

**Discussion by Lerr1uqs** [google/bumble#535] [Q&A] [answered]
1. Can bumble replace bluez as host layer to connect to the real bluetooth adaptor to send data package to other device?
2. Can bumble be seen as a host layer like bluez and fluoride that implements the logic in the Bluetooth specification?

**Comment by barbibulle** [comment, marked as answer]
1. Yes. Bumble can use real Bluetooth controllers. On all platforms, you can use a Bluetooth USB dongle or Bluetooth UART controller. On Linux, you can also use the hosts’s builtin Bluetooth controller if there is one. See the docs for details.
2. Yes, that’s exactly what Bumble does.
